src/Dialogs/BulkRenamer/RenamerListBox.vala

```diff
@@ -37,35 +37,25 @@ public class Files.RenamerListBox : Gtk.ListBox {
         construct {
             can_focus = false;
             var oldname_label = new Gtk.Label (old_name) {
-                xalign = 0.0f,
-                width_chars = 24
+                xalign = 0.0f
             };
 
 
             var newname_label = new Gtk.Label (new_name) {
-                xalign = 0.0f,
-                width_chars = 24
+                xalign = 0.0f
             };
 
             var arrow_image = new Gtk.Image.from_icon_name ("go-next-symbolic", Gtk.IconSize.MENU);
-            var ignored_image = new Gtk.Image.from_icon_name ("emblem-disabled", Gtk.IconSize.MENU) {
-                tooltip_text = _("Ignored - name is not changed")
-            };
-            var invalid_image = new Gtk.Image.from_icon_name ("emblem-error", Gtk.IconSize.MENU) {
-                tooltip_text = _("Cannot rename - name is invalid or already exists")
+
+            var status_image = new Gtk.Image () {
```

Comment:
We can use a single image and just update its properties to save ourselves some complexity here